python/paddle/optimizer/momentum.py

```diff
@@ -72,6 +73,7 @@ class Momentum(Optimizer):
             ( :ref:`api_fluid_clip_GradientClipByGlobalNorm` , :ref:`api_fluid_clip_GradientClipByNorm` ,
             :ref:`api_fluid_clip_GradientClipByValue` ). Default None, meaning there is no gradient clipping.
         multi_precision (bool, optional): Whether to use multi-precision during weight updating. Default is false.
+        use_multi_tensor (bool, optional): Whether to use multi-tensor strategy to update all parameters at once . Default is false.
```

Comment:
it should be listed after `rescale_grad`

Reply:
Done, tks!